test/jdk/jdk/jfr/event/oldobject/TestSanityDefault.java

```diff
@@ -37,7 +37,7 @@
  * @requires vm.hasJFR
  * @library /test/lib /test/jdk
  * @summary Purpose of this test is to run leak profiler without command line tweaks or WhiteBox hacks until we succeed
- * @run main/othervm jdk.jfr.event.oldobject.TestSanityDefault
+ * @run main/othervm -mx64m jdk.jfr.event.oldobject.TestSanityDefault
```

Comment:
Should it be `-Xmx`? I understand `-mx` is de jure standard, but we use `-Xmx` everywhere.

Reply:
I am thinking if 64M is a bit tight. For this test, we add `OldObjects.MIN_SIZE` (~100K) reachable objects in the list on every iteration. Ballparking each object having a marginal footprint of ~20 bytes, this means we allocate about 2M of data on every iteration. Suppose the previous iteration did not promote the object to old yet, but the very next allocation would. We would still need to absorb ~2M allocation into old, lest we run into OOM. Which means this thread implicitly relies on old being at least 2M, which might be not given for 64M heap. In practice, GCs would probably trash sooner.

So, maybe capping at more reasonable 1G would suffice to get reasonable test times without going into very tight heaps?

Reply:
The patch is updated as suggested